### ISSUE TITLE
Frontend: Use spinner as loading indicator by default

### ISSUE THREAD

**Issue by petertrr** [saveourtool/save-cloud#172]
Now we only use it in ProjectView. Can be implemented with [`Suspense`](https://reactjs.org/docs/concurrent-mode-suspense.html) in `App.kt`:
```kotlin
hashRouter {
    child(Suspense, props = jsObject{
        children =
    })
```

**Comment by akuleshov7** [comment]
Also need to create some preloading page while save is loading

**Comment by petertrr** [comment]
Implemented in #823, #830, #852 under #328
